src/pruner_util.cpp

```diff
@@ -0,0 +1,46 @@
+#include <fstream>
```

Comment:
Util contains all the helper functions we define. Moved all that stuff over here.